apps/system/lockscreen/js/lockscreen_state_manager.js

```diff
@@ -72,7 +72,9 @@
 
   LockScreenStateManager.prototype.start =
   function lssm_start(lockScreen) {
+    this.mainPromise = Promise.resolve();
```

Comment:
@timdream I've found this is necessary. Because if we don't have a queue, all following  events will trigger the next transferring rapidly (racing). So we need to queue these steps to make sure there is only one transferring is ongoing at one time.